src/main/kotlin/org/rust/ide/annotator/fixes/FillFunctionArgumentsFix.kt

```diff
@@ -64,8 +68,10 @@ class FillFunctionArgumentsFix(element: PsiElement) : LocalQuickFixAndIntentionA
 private fun getParameterTypes(element: PsiElement): List<Ty?>? {
     return when (element) {
         is RsCallExpr -> (element.expr.type as? TyFunction)?.paramTypes
-        is RsMethodCall -> (element.reference.resolve() as? RsFunction)?.valueParameterList?.valueParameterList?.map {
-            it.typeReference?.type
+        is RsMethodCall -> {
+            val substitution = element.inference?.getResolvedMethodSubst(element) ?: emptySubstitution
```

Comment:
I'd say, `getResolvedMethodType` is more applicable here

Reply:
You're right! Thanks.